bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/RecaptchaImpl.java

```diff
@@ -66,13 +79,11 @@ public class RecaptchaImpl extends AbstractFieldImpl implements Recaptcha {
 
     private static final String RECAPTCHA_URI = "uri";
 
-    @JsonIgnore
```

Comment:
size and cloud service path must not be part of json as they are part of custom props in the schema.
pls re-add @JsonIgnore for both props.